unit_test/test_spdm_requester/encap_digests.c

```diff
@@ -198,6 +207,98 @@ void test_spdm_requester_challenge_auth_case4(void **state)
 #endif
 }
 
+/**
+ * Test 5: receives a valid GET_DIGESTS request message from Requester
+ * Expected Behavior: produces a valid DIGESTS response message
+ **/
+void test_spdm_requester_challenge_auth_case5(void **state)
+{
```

Comment:
The case name is `test_spdm_requester_challenge_auth_case5`, but there is no challenge related function called in the test case. 
`test_spdm_requester_challenge_auth_case1`~`test_spdm_requester_challenge_auth_case4` already exist before we add this new test case, it is strange.